Tasks/ContainerStructureTestV0/task.json

```diff
@@ -0,0 +1,93 @@
+{
+    "id": "39bc2c9b-55b7-4835-89cd-6cc699ef7220",
+    "name": "ContainerStructureTest",
+    "friendlyName": "Container Structure Test",
+    "description": "Uses container-structure-test (https://github.com/GoogleContainerTools/container-structure-test) to validate the structure of an image based on four categories of tests - command tests, file existence tests, file content tests and metadata tests",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/test/publish-test-results",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=613742)",
+    "category": "Test",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "demands": [],
+    "minimumAgentVersion": "2.0.0",
+    "inputs": [
+        {
+            "name": "dockerRegistryServiceConnection",
+            "type": "connectedService:dockerregistry",
+            "label": "Container registry",
+            "groupName": "containerRepository",
+            "helpMarkDown": "Select a Docker registry service connection. Required for commands that need to authenticate with a registry."
+        },
+        {
+            "name": "repository",
+            "label": "Container repository",
+            "type": "string",
+            "helpMarkDown": "Name of the repository.",
+            "defaultValue": "",
+            "visibleRule": "command != login && command != logout",
+            "groupName": "containerRepository",
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "tags",
+            "type": "multiLine",
+            "defaultValue": "$(Build.BuildId)",
+            "properties": {
+                "resizable": "true",
+                "rows": "2"
+            },
+            "visibleRule": "command = build || command = push || command = buildAndPush",
+            "label": "Tags",
+            "groupName": "commands",
+            "helpMarkDown": "A list of tags in separate lines. These tags are used in build, push and buildAndPush commands. Ex:<br><br>beta1.1<br>latest"
+        },
+        {
+            "name": "image",
+            "type": "string",
+            "label": "Image",
+            "required": true,
+            "helpMarkDown": "Image to be tested"
+        },
+        {
+            "name": "configFile",
+            "type": "filePath",
+            "label": "Config Files",
+            "required": true,
+            "helpMarkDown": "Config files path, that contains container structure tests. Either .yaml or .json files"
+        },
+        {
+            "name": "failTaskOnFailedTests",
+            "type": "boolean",
+            "label": "Fail if there are test failures",
+            "defaultValue": "false",
```

Comment:
Is defaultValue: false consistent with how other test tasks run? I don't have much context on the test tasks, that's why I'm asking

Reply:
Yes. In other test tasks also we have it as false.